### DISCUSSION TITLE
The process randomly transcribes a duration limited audio file to return a block of text and now periods

### DISCUSSION THREAD

**Discussion by steelliberty** [openai/whisper#2173] [General]
I have been trying to use Whisper reliably for a web project, but the software is not cooperating. If I transcribe, say, 5 files over a 20-minute period (I doubt that has anything to do with the issue), maybe two out of the five will have no periods over most of the transcription or all of it.  If I take those same audio files and transcribe them again, they will add the periods properly.  I am using the base. en model but have the same issue with small.en and even medium but I am not interested in medium because it is too slow for my application.

I am using small audio files and some of the failures are files with duration of 8 minutes.

Does anyone have any clue as to why this might happen? 

Thank you, 

JM

**Comment by ryanheise** [comment]
See https://github.com/openai/whisper/discussions/194#discussioncomment-3766522
